test/PoolManager.t.sol

```diff
@@ -808,6 +808,47 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
         assertEq(manager.protocolFeesAccrued(currency1), expectedProtocolFee);
     }
 
+    function test_swap_toLiquidity_fromMinPrice() public {
```

Comment:
maybe worth naming this one `withLiquidity` and the other one `withoutLiquidity`?

Reply:
I think `toLiquidity` is accurate because this test traverses the swap until reaching nonzero liquidity. The relevant limit is the output amount, whereas in the other test the relevant limit is the price.

Reply:
(can still include `withoutLiquidity` in the other test name)